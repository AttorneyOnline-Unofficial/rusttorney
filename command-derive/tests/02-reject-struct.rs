use command_derive::*;

#[derive(Command)]
struct Handshake {
    hdid: String
}

fn main() {}
