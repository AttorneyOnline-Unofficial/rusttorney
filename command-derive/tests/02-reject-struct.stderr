error: expected `enum`
 --> $DIR/02-reject-struct.rs:4:1
  |
4 | struct Handshake {
  | ^^^^^^
